ups/ctrl_bps.table

```diff
@@ -4,5 +4,6 @@ setupRequired(daf_butler)
 setupRequired(pipe_base)
 setupOptional(ctrl_mpexec)
 
+envPrepend(CTRL_BPS_PLUGINS, $CTRL_BPS_DIR/python/lsst/ctrl/bps/cli/resources.yaml)
```

Comment:
you also should not need this file, for the same reasons I mentioned in the resources.yaml.